internal/characters/collei/skill.go

```diff
@@ -80,7 +80,9 @@ func (c *char) Skill(p map[string]int) action.ActionInfo {
 			duration += 180
 		}
 		c.AddStatus(sproutKey, duration, true)
-		c.QueueCharTask(func() { c.a1Ticks(src) }, sproutHitmark)
+		c.QueueCharTask(func() {
+			c.a1Ticks(src, nil)
```

Comment:
create the snapshot here and pass it to a1Ticks. make a1Ticks take combat.Snapshot directly instead of a pointer and remove the ``if snap == nil```. see kaeya q for example